ferveo-python/ferveo/__init__.pyi

```diff
@@ -119,6 +119,19 @@ class Dkg:
 
 @final
 class Ciphertext:
+    header: CiphertextHeader
+    payload: bytes
+
+    @staticmethod
+    def from_bytes(data: bytes) -> Ciphertext:
+        ...
+
+    def __bytes__(self) -> bytes:
+        ...
+
+
+@final
+class CiphertextHeader:
     @staticmethod
     def from_bytes(data: bytes) -> Ciphertext:
```

Comment:
presumably this is supposed to be:
```suggestion
    def from_bytes(data: bytes) -> CiphertextHeader:
```

Reply:
Hmm, curious that `mypy.stubtest` didn't catch that on CI.